warehouse/templates/includes/packaging/project-data.html

```diff
@@ -43,6 +43,26 @@ <h6>{% trans %}Maintainers{% endtrans %}</h6>
       </span>
     {% endfor %}
   {% endif %}
+  {% set verified_urls, unverified_urls = [],[] %}
+  {% for name, url in release.urls.items() %}
+    {% if is_valid_uri(url) %}
+      {% if name in ['Source', 'Homepage'] and project.is_verified_url(url) %}
+        {{ verified_urls.append((name, url)) }}
+      {% else %}
+        {{ unverified_urls.append((name, url)) }}
+      {% endif %}
+    {% endif %}
+  {% endfor %}
+  {% if release.github_repo_info_url and release.github_open_issue_info_url %}
```

Comment:
Where do these come from? This should probably be:

```suggestion
  {% if verified_urls.values() %}
```